packages/pretty-format/src/__tests__/prettyFormat.test.ts

```diff
@@ -209,21 +209,41 @@ describe('prettyFormat()', () => {
     expect(prettyFormat(val)).toEqual('123');
   });
 
+  it('prints a positive bigint', () => {
```

Comment:
can you wrap these tests in conditionals so theyæære not run if `BigInt` doesn't exist?